Sources/SpeziAccount/AccountValue/AccountKey+Views.swift

```diff
@@ -42,4 +42,20 @@ extension AccountKey {
     static func singleEditView(model: AccountOverviewFormViewModel, details accountDetails: AccountDetails) -> AnyView {
         AnyView(SingleEditView<Self>(model: model, details: accountDetails))
     }
+    
+    static func setupDisplayViewWithoutCurrentStoredValue(from details: AccountDetails) -> AnyView? {
+        guard let setupDisplayType = Self.DataDisplay as? any SetupDisplayView.Type else {
```

Comment:
This should be the correct way to compare the metatypes if I didn't miss anything.

```suggestion
        guard let setupDisplayType = Self.DataDisplay.self as? any SetupDisplayView.Type else {
```